zdm/survey.py

```diff
@@ -70,7 +71,7 @@ def get_efficiency_from_wlist(self,DMlist,wlist,plist,model="Quadrature"):
         self.mean_efficiencies=mean_efficiencies #be careful here!!! This may not be what we want!
         return efficiencies
     
-    def process_survey_file(self,filename):
+    def process_survey_file(self,filename, NFRB=None):
```

Comment:
Why do we want to tell it what NFRB is? Is the idea that we only wish to *use* NFRB out of a possibly greater number in the file? Can you you write down what the use of this parameter is?

Reply:
yes, you have that right.  Have added docs.